apps/fishing-map-e2e/src/e2e/map/map.cy.ts

```diff
@@ -0,0 +1,49 @@
+import { API_URL_4WINGS_TILES, API_URL_VESSELS, URL_FULL_DATA_AREA } from '../../constants/urls'
+import {
+  afterTestSkip,
+  beforeTestSkip,
+  disablePopups,
+  getTimeline,
+  waitForMapLoadTiles,
+} from '../../support/app.po'
+
+describe('map', () => {
+  before(() => beforeTestSkip())
+  afterEach(afterTestSkip)
```

Comment:
@fertandil87 Why are we skipping before and after tests calls?

Reply:
@rdgfuentes It was a proposal and it was merged... This was already discussed and exist another PR to fix it https://github.com/GlobalFishingWatch/frontend/pull/2147